OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -453,6 +453,12 @@ private void UpdateStatus(ShortApplicationDto applicationDto, ApplicationDto app
             }
         }
 
+        if (application.Workshop.CompetitiveSelection && application.Status == ApplicationStatus.Pending &&
```

Comment:
Please add tests for different scenarios.

Reply:
Tests added